app/services/form1010cg/service.rb

```diff
@@ -85,7 +85,7 @@ def submit_attachment # rubocop:disable Metrics/MethodLength,Metrics/CyclomaticC
 
         carma_attachments.submit!(carma_client)
         submission.attachments = carma_attachments.to_hash
-      rescue
+      rescue => e
```

Comment:
Is there a specific exception that you would want to rescue from here? Catch-all rescues may swallow up bugs that are unrelated, so specifying `Common::Client::Error` or whatever you'd expect to see error here, you might want to specify.

Reply:
In this case, I want to capture **all** exceptions. This method should only return true and false without raising an error. Mainly to prevent the error from bubbling up to the response and making user's thing their submission did not go through.

But I could also see the argument of the #new, #add, and #attachments= being moved out of the catch block since it doesn't connect to anything externally, it's just source code.

Does `Common::Client::Error` capture all network errors? I'd probably need to catch all 4** and 5** errors.